src/netex_idf/offers.rs

```diff
@@ -60,7 +111,8 @@ pub fn read_offer_folder(offer_folder: &Path, _collections: &mut Collections) ->
             .parse()
             .map_err(|_| format_err!("Failed to open {:?}", offer_path))?;
         info!("Reading {:?}", offer_entry.path());
-        parse_offer(&offer)?;
+        let routes = parse_offer(&offer, collections)?;
```

Comment:
let routes = skip_fail!(parse_offer(&offer, collections).map_err(|e| format_err!(
    "Skip file {}: {}",
    offer_path.display(),
    e
)));
Netex files will be dirty for a long time. 
It's better not to fail at the first invalid feed.

Reply:
I now skip the invalid routes and I added some fixtures for it in 62243cb0d655c2de74036b90559ec6f7055503bf.